src/saga/login.js

```diff
@@ -186,6 +187,7 @@ function* autoConnectCheck () {
 }
 
 function* initialLoad () {
+  yield put(getUserGroups())
```

Comment:
These `put`s run in parallel -- may want to block on fetch of the user group to ensure they're loaded before processing permissions, editability, visibility based on user/group permissions...